docker/build.Dockerfile

```diff
@@ -76,3 +88,12 @@ RUN  CXX_COMPILER="g++" \
     && cmake --build . --target check \
     && cmake --build . --target install
 
+# Install Vulkan SDK
+RUN wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | apt-key add - \
+    && wget -qO /etc/apt/sources.list.d/lunarg-vulkan-1.3.216-bionic.list https://packages.lunarg.com/vulkan/1.3.216/lunarg-vulkan-1.3.216-bionic.list \
+    && apt-get update \
+    && apt-get install -yqq --no-install-recommends \
+       vulkan-sdk x11-xserver-utils libvulkan-dev libvulkan1 xvfb mesa-vulkan-drivers
```

Comment:
I think it'd be better to install everything in the first `RUN` command. This way we can fail early when the installation fails for whatever reason.

Reply:
Also, doesn't ubuntu come with some default packages for vulkan? Maybe we could skip the step where we add the key and download the list from lunarg?

Reply:
Ubuntu doesn't come with default packages for Vulkan. If I remove the step containing adding the key and downloading the list, it will be unable to locate vulkan-sdk and the installation fails.